site/src/docs/components/breadcrumb/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+slug: '/components/breadcrumb'
+title: 'Breadcrumb'
+navTitle: 'Breadcrumb'
+---
+
+import { IconDocument, IconPhone, IconEnvelope, IconPhoto, Breadcrumb } from 'hds-react';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<Breadcrumb
```

Comment:
This is missing the PlaygroundPreview component around it.

Reply:
Wrapped.